tensorflow_serving/model_servers/tensorflow_model_server_test.py

```diff
@@ -122,7 +122,8 @@ def RunServer(self,
                 model_name,
                 model_path,
                 batching_parameters_file='',
-                wait_for_server_ready=True):
+                wait_for_server_ready=True,
```

Comment:
nit: please swap the order of these args s.t. wait_for_server_ready remains the last arg (and update callsites as needed)